package.json

```diff
@@ -79,16 +79,15 @@
     "@storybook/react": "^3.4.1",
     "bootstrap": "^3.3.7",
     "classnames": "^2.2.5",
-    "font-awesome": "^4.7.0",
     "hex-rgb": "^1.0.0",
     "jquery": "^3.2.1",
+    "js-yaml": "^3.12.0",
```

Comment:
The weight of that library is huge: https://bundlephobia.com/result?p=js-yaml@3.12.0

Is there any other option?

Reply:
[yamljs](https://bundlephobia.com/result?p=yamljs@0.3.0) is the lightest i've found.
Would that be ok ?

Reply:
I'm afraid we'll have to do it another way... 171 kB is too huge for a low-level module IMO.

As we can see in the `src/components/IconPicker/IconPicker.jsx` file, the FontAwesome version is related to the OSM UI version. I mean, for a build of OSM UI there is a version of FontAwesome.

So what about this:
* We create a script to download the YAML file and convert it into a JSON format
* At the build time of OSM UI, we launch that script
* At the run time, we use the JSON file

That way we have the same level of feature and less weight to carry :)

P.S.: For the launch of the new script, you have the choice of creating a Makefile or using the `pre` prefix in the `package.json` file

```
"prebuild": "download and convert YAML",
"build": "same command as before",
```

Reply:
The data i need as JSON is 49kB, will that be ok ?

Reply:
It's more than 3 times lighter than the yaml module itself, and we don't need to process/convert the yaml file at the runtime 👍